openapi.yaml

```diff
@@ -132,147 +467,202 @@ securityDefinitions:
     scopes:
       full: Full access to the API
 definitions:
-  Meta:
+  PizzasResults:
     properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
-  SelfLink:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/PizzaResource"
+  PizzaResult:
     properties:
-      self:
```

Comment:
Although optional, almost all of our APIs respond with a top-level self link. Could you add one here?

Reply:
missed in this in the previous round of reviews; added to dough and ingredient response objects as well